coq-autosubst-examples.opam

```diff
@@ -0,0 +1,14 @@
+opam-version: "2.0"
+maintainer: "palmskog@gmail.com"
+version: "dev"
+
+homepage: "https://github.com/coq-community/autosubst"
+dev-repo: "git+https://github.com/coq-community/autosubst.git"
+bug-reports: "https://github.com/coq-community/autosubst/issues"
+license: "MIT"
+
+synopsis: "Autosubst Examples (CI only)"
+
+build: [make "-j%{jobs}%" "examples-plain"]
+install: []
+depends: ["coq-autosubst"]
```

Comment:
Actually this will just build the lib due to the makefile dependency, so this dependency here makes little sense...